_includes/v20.2/backups/backup-options.md

```diff
@@ -2,5 +2,5 @@
 -----------------------------------------------------------------+-------------------------+------------------------------
 `revision_history`<a name="with-revision-history"></a>           | N/A                     | Create a backup with full [revision history](take-backups-with-revision-history-and-restore-from-a-point-in-time.html), which records every change made to the cluster within the garbage collection period leading up to and including the given timestamp.
 `encryption_passphrase`<a name="with-encryption-passphrase"></a> | [`STRING`](string.html) | The passphrase used to [encrypt the files](take-and-restore-encrypted-backups.html) (`BACKUP` manifest and data files) that the `BACKUP` statement generates. This same passphrase is needed to decrypt the file when it is used to [restore](take-and-restore-encrypted-backups.html) and to list the contents of the backup when using [`SHOW BACKUP`](show-backup.html). There is no practical limit on the length of the passphrase.
-`DETACHED`                                                       | N/A                     | <span class="version-tag">New in v20.2:</span> When a backup runs in `DETACHED` mode, it will execute asynchronously and the job ID will be returned immediately without waiting for the job to finish. Note that with `DETACHED` specified, further job information and the job completion status will not be returned. For more on the differences between the returned job data, see the [example](backup.html#run-a-backup-asynchronously) below. To check on the job status, use the [`SHOW JOBS`](show-jobs.html) statement. <br><br>To run a backup within a [transaction](transactions.html), use the `DETACHED` option.
+`DETACHED`                                                       | N/A                     | <span class="version-tag">New in v20.2:</span> When a backup runs in `DETACHED` mode, it will execute asynchronously. The job ID will be returned after the backup is validated. Note that with `DETACHED` specified, further job information and the job completion status will not be returned. For more on the differences between the returned job data, see the [example](backup.html#run-a-backup-asynchronously) below. To check on the job status, use the [`SHOW JOBS`](show-jobs.html) statement. <br><br>To run a backup within a [transaction](transactions.html), use the `DETACHED` option.
```

Comment:
Hey Ashley, my bad, but thinking on this... We have just added some "backup validation" to the DR feature set. These are for checking that a backup is restorable. You can see the docs for those here: https://www.cockroachlabs.com/docs/dev/backup-validation.html. This is completely different to what we are talking about in this PR.

As a result, I think that the word "validated" in this PR should be changed. Since, we're using that link to the backup arch docs in most places, a good replacement would be "after the backup's job creation phase completes." Or "after backup job creation completes"... whichever you think sounds better.

So, switching out:

"The job ID will be returned after the backup is validated."

For, something along the lines of:

"The job ID will be returned after the backup job creation completes."

(With the link for the relevant versions). The same phrasing can be used for restore here.